apps/web/src/components/FiatOnRampModal/FiatOnRampModal.tsx

```diff
@@ -87,11 +88,16 @@ interface FetchResponse {
   urlWithSignature: string
 }
 
+function generateRandomString(length) {
+  const randomBytes = crypto.randomBytes(length)
```

Comment:
maybe crypto instance?
https://developer.mozilla.org/en-US/docs/Web/API/crypto_property

Reply:
no just need to generate random string. will actually use uuid. just orgnlly used crypto to save the exttra package

Reply:
removed